lms/djangoapps/certificates/api.py

```diff
@@ -228,8 +229,13 @@ def certificate_downloadable_status(student, course_key):
         'is_unverified': True if current_status['status'] == CertificateStatuses.unverified else False,
         'download_url': None,
         'uuid': None,
+        'is_invalid_cert': False,
```

Comment:
@schenedx This function cert_generation_enabled is just for checking whether certificate generation is enabled for a course and here we don't have any student object and his certificates list. With this change we have to hit the 'generated_cert' and 'invalidated_certs' each time when user loads the progress page.
Advantage with current implementation is we already have status of certificate and in case on 'un-available' status only then I hit the db to see invalidated status to avoid extra queries.